### DISCUSSION TITLE
Support flutter 3.13.0

### DISCUSSION THREAD

**Discussion by rustdesk** [fzyzcjy/flutter_rust_bridge#1326] [Q&A]
I am migrating from 1.75.3 to 1.81.0, because 1.75.3 does not work with flutter 3.13.0 , below command failed with build_runner. 

```
~/.cargo/bin/flutter_rust_bridge_codegen --rust-input ../src/flutter_ffi.rs --dart-output ./lib/generated_bridge.dart --c-output ./macos/Runner/bridge_generated.h
```

<= 1.80.0 generated correct bridge rs file, but build_runner failed.
1.81.1 generated almost empty bridge rs file, though build_runner ok.

**Comment by fzyzcjy** [comment]
Hmm it looks like a bug. Feel free to create an issue!

It would be great to provide a minimal reproducible sample as well as logs (following the issue template)
